shared/fs/row.js

```diff
@@ -24,13 +25,14 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({
     if (type === 'folder') {
       dispatch(navigateAppend([{props: {path}, selected: 'folder'}]))
     } else {
-      dispatch(FsGen.createDownload({path}))
+      dispatch(FSGen.createDownload({path}))
       console.log('Cannot view files yet. Requested file: ' + Types.pathToString(path))
     }
   },
+  _openInFileUI: (path: Types.Path) => dispatch(FSGen.createOpenInFileUI({path: Types.pathToString(path)})),
```

Comment:
After rebasing to this branch, I'm realizing that this is a plain "open" call, which I guess is why you filtered on `type === folder` in the component. It seems a "show in Finder" button probably makes more sense, especially for files?

Reply:
I don't follow...for folders, we open the target path. For files, we `shell.showItemInFolder`.

Reply:
> we open the target path. For files, we shell.showItemInFolder.

That sounds good with me. (I feel like we talked about this in a meeting but couldn't remember clearly.) Currently in this PR we only show icon for folders though. But I'm OK with adding it later too; just thought I'd bring it up.

Reply:
Ah yeah. I didn't want to step on your toes re: clicking files. We can add it once your downloading code is merged.